main.cpp

```diff
@@ -885,6 +885,9 @@ int main(int argc, char ** argv) {
         antipromptv_inp.push_back(::llama_tokenize(vocab, antiprompt, false));
     }
 
+    // tokenize the first reverse prompt
+    std::vector<llama_vocab::id> first_antiprompt = ::llama_tokenize(vocab, params.antiprompt.front(), false);
```

Comment:
If `params.antiprompt` is empty, this will crash. Better check first.

```
std::vector<llama_vocab::id> first_antiprompt;
if (!params.antiprompt.empty()) {
    first_antiprompt = ::llama_tokenize(vocab, params.antiprompt.front(), false);
}
```

Reply:
On it. Thanks